src/main/java/com/checkout/AbstractCheckoutSdkBuilder.java

```diff
@@ -1,12 +1,14 @@
 package com.checkout;
 
 import java.net.URI;
+import java.util.concurrent.Executor;
 
 public abstract class AbstractCheckoutSdkBuilder<T extends CheckoutApiClient> {
 
     private Environment environment;
     private URI uri;
     private CheckoutFilesApiConfiguration filesApiConfiguration = CheckoutFilesApiConfiguration.disabled();
+    private Executor executor;
```

Comment:
`Executor` or `ExecutorService`, wdyt?

Reply:
The reason of use `Executor` is because is the parent of `ExecutorService`, so clients can provide either an Executor implementation or an ExecutorService implementation, given that we are not using anything else about eh thread pool.